app/views/events/_show_event_large.html.erb

```diff
@@ -1,46 +1,62 @@
 <%- model_class = Event -%>
-<div>
-  <div class="pull-right">
+<div class="row relative-box">
+  <div class="col-xs-12 col-sm-9 col-md-9 col-lg-9">
+    <h1 class="page-title"><%= @event.name %></h1>
+  </div>
+  <div class="col-xs-12 col-sm-3 col-md-3 col-lg-3 text-right bottom-align-text">
     <%= render :partial => 'apply_button', :locals => {event: @event} %>
     <%= link_to t('.edit', :default => t('helpers.links.edit')), edit_event_path(@event),
                 :class => 'btn btn-default btn-sm' if can? :edit, @event %>
   </div>
-  <h1><%= @event.name %></h1>
 </div>
 <div class="row">
-  <div class="col-md-1 col-md-offset-1 text-center col-sm-2">
+  <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 line-spacer">
+  </div>
+</div>
```

Comment:
What is this whole row? Just for spacing? Use a single `<div>` without class instead.